packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -83,20 +92,22 @@ export async function setActiveAccount(
     if (response.data.status !== 'Success') {
       throw Error(`Unsuccessful status: ${response.status}`);
     }
-  } catch (err) {
-    console.error(err?.response.data.error || err?.message);
-    notifyError('Could not set active account');
-  }
 
-  app.user.ephemerallySetActiveAccount(account);
-  if (
-    app.user.activeAccounts.filter((a) => isSameAccount(a, account)).length ===
-    0
-  ) {
-    app.user.setActiveAccounts(
-      app.user.activeAccounts.concat([account]),
-      shouldRedraw,
-    );
+    app.user.ephemerallySetActiveAccount(account);
+    if (
+      app.user.activeAccounts.filter((a) => isSameAccount(a, account))
+        .length === 0
+    ) {
+      app.user.setActiveAccounts(
+        app.user.activeAccounts.concat([account]),
+        shouldRedraw,
+      );
+    }
+  } catch (err) {
+    // Failed to set the user's active address to this account.
+    // This might be because this address isn't `verified`,
+    // so we don't show an error here.
+    console.error(err?.response?.data?.error || err?.message);
```

Comment:
There is no indication to the user that something went wrong, lets keep the notification

Reply:
Cool, yeah I put this back.